lib/provisioning/account.js

```diff
@@ -0,0 +1,292 @@
+const call_account_api = require('../network/call_account_api');
+
+/**
+ * @desc - List all sub_accounts under the account.
+ * @param [enabled] {boolean} - Whether to only return enabled sub-accounts (true) or disabled accounts (false). Default: all accounts are returned (both enabled and disabled).
+ * @param [ids] {number[]} - A list of up to 100 sub-account IDs. When provided, other parameters are ignored.
+ * @param [prefix] {string} - Returns accounts where the name begins with the specified case-insensitive string.
+ * @param [options] {object} - Generic advanced options map, see online documentation.
+ * @param [callback] {function}
+ */
+function sub_accounts(enabled, ids = [], prefix, options = {}, callback) {
+  let params = {
+    enabled,
+    ids,
+    prefix,
+  };
+
+  let uri = ['sub_accounts'];
+  return call_account_api('GET', uri, params, callback, options);
+}
+
+
+/**
+ * @desc Get a specific sub_account by sub_account_id
+ * @param sub_account_id {string} - The ID of the sub-account to get.
+ * @param [options] {object} - Generic advanced options map, see online documentation.
+ * @param [callback] {function}
+ */
+function sub_account(sub_account_id, options = {}, callback) {
+  let uri = ['sub_accounts', sub_account_id];
+  return call_account_api('GET', uri, {}, callback, options);
+}
+
+
+/**
+ *
+ * @param name {string} The display name as shown in the management console.
+ * @param cloud_name {string} A case-insensitive cloud name comprised of alphanumeric and underscore characters. Use with caution - generates an error if the cloud name is not unique across all Cloudinary accounts. Default: a unique cloud name automatically generated by Cloudinary.
+ * @param custom_attributes {object} Any custom attributes you want to associate with the sub-account, as a map/hash of key/value pairs.
+ * @param enabled {boolean} Whether the sub-account is enabled. Default: true
+ * @param base_account {string} The ID of another sub-account, from which to copy all of the following settings: Size limits, Timed limits, and Flags.
+ * @param options
```

Comment:
Options description is not consistent across functions, please align

Reply:
Fixed